exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/Foreman.java

```diff
@@ -417,6 +417,9 @@ private void parseAndRunPhysicalPlan(final String json) throws ExecutionSetupExc
 
   private void runPhysicalPlan(final PhysicalPlan plan) throws ExecutionSetupException {
     validatePlan(plan);
+    //Marking endTime of Planning
+    queryManager.markPlanningEndTime();
```

Comment:
Right. The method actually is multiple steps..
1. Validate Plan
2. SortMemAllocations
3. Wait In Queue
4. Setup Fragments
5. Execute... 

I wanted to explicitly provide the delta time until step 1. 
Step 2 is not related to planning (though we can consider including it). Logically, it is planning. 
Step 3 is probably useful only in the RM scenario, but I cannot estimate it for older profiles. Let's file a **separate** JIRA for this bit? It most certainly involves another addition to the protobuf. 
Step 4. Expected to be a low cost operation like Step 2 and not certainly related to planning.